README.md

```diff
@@ -123,19 +123,26 @@
   - [mydumper](tools/mydumper.md)
   - [Loader](tools/loader.md)
   + Data Migration
-    + Overview
-      - [Architecture](tools/data-migration-overview.md#architecture)
-      - [Features](tools/data-migration-overview.md#data-synchronization-introduction)
-      - [Restrictions](tools/data-migration-overview.md#usage-restrictions)
+    - [Overview](tools/data-migration-overview.md)
+    - [Restrictions](tools/data-migration-overview.md#usage-restrictions)
+    + Features
+      - [Table Routing](tools/dm-data-synchronization-features.md#table-routing)
+      - [Black and White Lists](tools/dm-data-synchronization-features.md#black-and-white-table-lists)
+      - [Binlog Event Filtering](tools/dm-data-synchronization-features.md#binlog-event-filtering)
+      - [Column Mapping](tools/dm-data-synchronization-features.md#column-mapping)
+      - [Synchronization Delay Monitoring](tools/dm-data-synchronization-features.md#synchronization-delay-monitoring)
+      + Sharding Support
+        - [Introduction](tools/dm-sharding-solution.md)
+        - [Restrictions](tools/dm-sharding-solution.md#sharding-ddl-usage-restrictions)
+        - [Troubleshoot](tools/troubleshooting-sharding-ddl-locks.md)
+    + Usage Scenarios
+      - [Simple Scenario](tools/dm-simple-synchronization-scenario.md)
+      - [Shard Merge Scenario](tools/dm-shard-merge-scenario.md)
     - [Deploy](tools/data-migration-deployment.md)
-    - [Synchronize Data](tools/data-migration-practice.md)
+    - [Synchronize](tools/data-migration-practice.md)
```

Comment:
@csuzhangxc Do you have any suggestions on this? :)

Reply:
*Get Started*? 😂

Reply:
Haha~ Thanks for your suggestion, but I think it might be a little bit weird to use "Get Started" in this position. @GregoryIan What's your opinion and suggestion?

Reply:
it's a step by step instructions using dm-ansible

Reply:
dmctl commands also used in it.

Reply:
@GregoryIan The "Deploy" file is a step-by-step instruction using DM-Ansible. The "Synchronize" file is a step-by-step instruction about how to synchronize data.

Reply:
I want to add a step by step instrucation that contains deploy dm manually and create a simple task, like this file do

Reply:
@GregoryIan Perhaps just leave a `Deploy` in the TOC that points to the `tools/data-migration-practice.md` file? That file about deploying the DM cluster using DM-Ansible is also mentioned in this file.

Reply:
ok

Reply:
how about
```
[get started](tools/data-migration-practice.md)
Deploy
- [ansible](tools/data-migration-deployment.md)
- [binary](xxxxx)
```
What I can confirm is that we have to need a `get started`. If the content of `tools/data-migration-practice.md` is not a good enough, we can modify it.